src/utils/json.ts

```diff
@@ -53,11 +40,11 @@ export function tryParseJson(json, onFailed) {
 /**
  * Converts the passed string into a JSON object (if possible) and returns the
  * value of the propertyName on that object.
- * @param {string} jsonString
- * @param {string} propertyName
- * @return {*}
  */
-export function getPropertyFromJsonString(jsonString, propertyName) {
+export function getPropertyFromJsonString(
+  jsonString: string,
+  propertyName: string
+): any {
   const json = tryParseJson(jsonString);
   return (json && json[propertyName]) || null;
```

Comment:
Doesn't look like this handles falsey values correctly. For example, it will return `null` for `getPropertyFromJsonString('{"value": false}', 'value')`. Can you add a TODO to fix this (separately from TS migration)?

Reply:
Added 👍